### PR TITLE
Create Video Player Page

### LINKED ISSUE #43: Create video player page
### Acceptance Criteria

1. Player page must allow a user to view a single video
1. Player page must be designed according to previously-made mockups (loosely)

### PR DESCRIPTION
**Pull request by tanner-g**
# Acceptance Criteria
Player page must allow a user to view a single video
Player page must be designed according to previously-made mockups (loosely)

Closes #43

### CODE REVIEW

**Comment by tanner-g** [comment]
The travis test said that it failed because of a missing final newline. Not sure what thats about so I just added a newline. Might be a python thing.

**Comment by tanner-g** [comment]
Functionality: 
- Users can now navigate to a 'video player' page and back the manage videos page.

**Comment by tanner-g** [comment]
Looking into tests to add.

File: src/templates/video_player.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    </a>
+    <div class="user-buttons">
+      <a href="/">
+        <button class="logout-button" type="button">Manage</button>
+      </a>
+      <a href="/logout">
+        <button class="logout-button" type="button">Logout</button>
+      </a>
+    </div>
+    <!-- needs a button for main page-->
```
Comment (JasonPuglisi) [inline src/templates/video_player.html]: Emmental logo goes to the main page

File: src/templates/video_player.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          <source src="/video/{{ video.video_id }}.{{ video.extension }}" type="video/ogg">
+          Your browser does not support HTML5 Video.
+        </video>
+    </div>
+    {% else %}
+    <h2>Video not found.</h2>
+    {% endif %}
+  </div>
+</section>
+{% endblock %}
```
Comment (JasonPuglisi) [inline src/templates/video_player.html]: Newline at end of files is standard in a bunch of languages which is why the build earlier failed. Can you add one here?

File: src/templates/video_player.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      {% endfor %}
+    </div>
+    {% endif %}
+    {% endwith %}
+
+    <h1>Video Player:</h1>    
+    {% if video %}
+    <div class="videos">
+        <video width="75%" controls>
+          <source src="/video/{{ video }}" type="video/mp4">
```
Comment (JasonPuglisi) [inline src/templates/video_player.html]: Change type to `type="video/{{ video.extension }}"`

File: src/templates/video_player.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          <source src="/video/{{ video.video_id }}.{{ video.extension }}" type="video/ogg">
```
Comment (JasonPuglisi) [inline src/templates/video_player.html]: Remove this line

File: src/templates/content.html [hunk 5]
```diff
@@ -25,14 +25,14 @@
     {% endif %}
     {% endwith %}
 
-    <h1>Video Index</h1>
+    <h1>Manage Uploads</h1>
```
Comment (JasonPuglisi) [inline src/templates/content.html]: Manage uploads as a title doesn't make much sense to me as this page will show videos uploaded from all users.

File: src/templates/video_player.html (hunk 1 above)
Comment (tanner-g) [reply src/templates/video_player.html]: Yes. I need to remove that comment. I ended up making a button that goes back to the 'video index' page. Needed a way to back to previous page.

File: src/templates/video_player.html (hunk 2 above)
Comment (tanner-g) [reply src/templates/video_player.html]: yup

File: src/templates/video_player.html (hunk 3 above)
Comment (tanner-g) [reply src/templates/video_player.html]: Yeah. I only need to path to the video link. So, that's all I pass.

File: src/templates/video_player.html (hunk 4 above)
Comment (tanner-g) [reply src/templates/video_player.html]: the whole thing?

File: src/templates/content.html (hunk 5 above)
Comment (tanner-g) [reply src/templates/content.html]: I think that it makes it look more like the renders. I agree that its an index of all videos and not just ones uploaded from the current user.

Comment (tanner-g) [reply src/templates/content.html]: I'll change it back.

**Comment by tanner-g** [comment]
Changes:
- extra new line added
- 'manage uploads' to 'video index'
- removed dangling comment

**Comment by tanner-g** [comment]
Changes:
- video extensions are now dynamically added( doesn't have extra line)

**Pull request merged**